src/servers/shared_memory_block_manager.cc

```diff
@@ -36,10 +36,21 @@ SharedMemoryBlockManager::~SharedMemoryBlockManager()
   }
 }
 
+#if TRTIS_ENABLE_GPU
```

Comment:
Instead of making a super-set Create method, it is more maintainable and easier to read if you make 2 Create methods, one for each case.

Reply:
Done